lib/proxy/util.rb

```diff
@@ -18,7 +18,6 @@ def start(&ensured_block)
       # run the task in its own thread
       logger.debug "Starting task: #{@command}"
       @task = Thread.new(@command) do |cmd|
-        begin
           status = nil
```

Comment:
The indenting probably also needs to be modified, right?

Reply:
Should get auto-corrected automatically, after GH-723 and GH-724 got in.